tests/generate_fixtures/generate_pyani_anim_matrices.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# The MIT License
+#
+# Copyright (c) 2024 University of Strathclyde
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+#create db
+pyani createdb --force
+
+#index with pyANI v0.3
+pyani index -i ../../tests/fixtures/viral_example/
+
+#Update the label file so that the columns in the matrices correspond to the genome stem files
+awk -F'\t' 'OFS="\t" {$3 = $2; print}' ../../tests/fixtures/viral_example/labels.txt > ../../tests/fixtures/viral_example/labels_tmp.txt && mv ../../tests/fixtures/viral_example/labels_tmp.txt ../../tests/fixtures/viral_example/labels.txt
```

Comment:
Can we do something clever here to insert the MD5 checksums as the labels?

Perhaps using the output of ``md5sum ../../tests/fixtures/viral_example/*.f*``?

Reply:
Changing `{$3 = $2; print}` to `{$3 = $1; print}` fixed this issue. The first column in `labels.txt` already provides md5sum :)

Reply:
Easier than I expected. How does this look with the new output?